ChangeLog.md

```diff
@@ -204,6 +204,8 @@
   * Remove old temporary upgrade flag `tmp_upgraded_iq_status` (Matt West).
 
   * Remove `string_from_2darray_sf()` from `freeformPythonLib/prairielearn.py` (Liz Livingston)
+  
+  * Fix hljs syntax highlighting compatibility issue in `pl-code.py` (Eric Huber)
```

Comment:
Nit: changelog entries should be grouped by prefix ("fix") and end with a period

Reply:
OK, recommitted that adjustment